app/views/tmdb/actor_search.html.erb

```diff
@@ -2,17 +2,21 @@
 
 <h1>Search by Actor</h1>
 <% if @results&.actor.blank? %> <!-- Display the search form -->
-  <% if @results&.not_found_message.present? %>
-    <h2>Hmm. <%= @results.not_found_message %></h2>
-    <p>Check your spelling and try again.</p>
-  <% end %>
   <%= form_tag '/tmdb/actor_search', { class: "form-class", id: "actor-search", role: "form", method: :get } do %>
     <%= text_field_tag :actor, nil, class: "form-control search-form autocomplete-search-field", id: "actor_name_actor_search", placeholder: "Enter Actor Name", data: { autocomplete_source: person_autocomplete_path } %>
     <%= select_tag :sort_by, options_for_select(Movie::SORT_BY), {:prompt => "Sort by...", class: "form-control search-form", :include_blank => false } %>
     <%= submit_tag "Search", id: "submit_button_actor_search", class: "form-control-submit search-form-submit" %>
   <% end %>
+<% end %>
+
+<% if @results&.not_found_message.present? %>
```

Comment:
This view change is not related to this PR, but it was a bug and I decided to fix it anyway. 😬